frontend/src/pages/events/index.js

```diff
@@ -4,11 +4,22 @@ import Layout from "../../components/layout";
 import Seo from "../../components/seo";
 import Instagram from "../../components/instaBar";
 import SundayCelebBarEvents from "../../components/page-events/sundayCelebBarEvents";
+import PrayerGathering from "../../images/prayer-gathering.png";
+import EventCard from "../../components/page-events/eventCard";
 
 const EventsPage = () => (
   <Layout>
     <SundayCelebBarEvents />
     <div>Event list section</div>
+    <EventCard
+      title={"Prayer Gathering"}
+      date={"Th, May 25, 2023"}
+      img={PrayerGathering}
```

Comment:
use relative path for the use of StaticImage
```suggestion
      img={../../images/prayer-gathering.png}
```